docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  app:
+    container_name: reviews-service
+    restart: always
+    build: .
+    ports:
+      - '3001:3001'
```

Comment:
This is where you set your port at 3001, but where did you set your path variable API_URL?

Reply:
If you are referring to localhost vs IP of the EC2 instance I'm using, I'm setting that environmental variable inside the package.json file.